### ISSUE TITLE
Fails to recognize missing colon (,) in arrays

### ISSUE THREAD

**Issue by veteran29** [SkaceKamen/vscode-sqflint#35]
This does not produce error:
```sqf
params [
    ["_start", nil, [[]], 3],
    ["_end", nil, [[]], 3],
    ["_focus", nil, [objNull, []], 3]   // <-- missing "," does not result in syntax error
    ["_camera", nil, [objNull]]
];
```

Additional colon is properly seen as error:
```sqf
params [
    ["_start", nil, [[]], 3],
    ["_end", nil, [[]], 3],
    ["_focus", nil, [objNull, []], 3],
    ["_camera", nil, [objNull]], // <-- produces error
];
```

```sqf
private _arr = ["1" "2", "3"]; // no error
private _arr = ["1", "2" "3"]; // no error
private _arr = ["1", "2" "3",]; // error
```

**Comment by SkaceKamen** [comment]
This is because the "missing" colon is hard to recognise in current grammar, since I was lazy and the grammar is very broad. Any two expressions next to each other are allowed.

In the SQF something like this is valid with or without the colon:
```sqf
private _arr1 = [1, player, call BIS_fnc_something];
private _arr2 = [1, player call BIS_fnc_something];
```
so, as far as the grammar is concerned right now, it's the same as
```sqf
private _arr = [1, 2 3, 4];
```
(the `3` is argument of command `2`)

I know this is terrible. I'll need to create some kind of inteligent check whenever two expressions next to each other are callable or not. I'm thinking about making all the "operators" (commands) part of grammar, so the grammar correctly recognises these issues... that may take a while

And as far as the trailing colon, since when is that allowed? I feel like when I was making this extension, it produced error.

**Comment by veteran29** [comment]
Trailing colon is not allowed ("Additional colon is __properly__ seen as error:") I've just added it as an example that it works correctly in that case.

**Comment by SkaceKamen** [comment]
I'm blind, sorry :'(.

**Comment by SkaceKamen** [comment]
Partially fixed in version 0.9.0
